packages/render-fidelity-tools/test/config.json

```diff
@@ -394,7 +394,8 @@
       "orbit": {
         "radius": 12
       },
-      "renderSkybox": true
+      "renderSkybox": true,
+      "realTimeFiltering": true
```

Comment:
This scenario doesn't need realTimeFiltering because it is using the default IBL, which is not especially HDR.